src/handlers/events/delivery/goals/httpServiceGoals.ts

```diff
@@ -20,12 +20,18 @@ export const ReviewGoal = new Goal({
     completedDescription: "Code review passed",
 });
 
-export const BuildGoal = new Goal({
+export const CodeReactionGoal = new Goal({
+    environment: IndependentOfEnvironment,
+    orderedName: "1.5-react",
+    completedDescription: "Code reactions",
+});
+
+export const BuildGoal = new GoalWithPrecondition({
     environment: IndependentOfEnvironment,
     orderedName: "2-build",
     workingDescription: "Building...",
     completedDescription: "Build successful",
-});
+}, AutofixGoal);
```

Comment:
build now waits on Autofix, cool

Nothing waitts on CodeReaction (fine)
Notthing waits on Scan -- is that intended?